EventListener/TemplateListener.php

```diff
@@ -103,12 +103,12 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             };
 
             $event->setResponse(new StreamedResponse($callback));
+        } else {
+            $event->setResponse($templating->renderResponse($template->getTemplate(), $parameters));
         }
 
         // make sure the owner (controller+dependencies) is not cached or stored elsewhere
         $template->setOwner(array());
```

Comment:
You either want a streamed response or a regular response. When a streamed response was chosen it was simply overwritten a few lines later on anyway. Putting an else there makes sure it won't overwrite anymore.